### PR TITLE
TWAP module internal audit comments

### PR DESCRIPTION
**Pull request by AlpinYukseloglu**
Closes: #2480 

## What is the purpose of the change

This PR adds clarifying comments to areas that took some digging around to fully understand during internal review of the TWAP module

## Brief Changelog

N/A

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not applicable)

### CODE REVIEW

**Review by czarcas7ic — APPROVED** [review]

File: x/twap/api.go
Comment (ValarDragon) [inline x/twap/api.go:85]:
```suggestion
```

File: x/twap/api.go
Comment (AlpinYukseloglu) [reply x/twap/api.go:32]: Right right great point!

**Review by mattverse — APPROVED** [review]
Good to merge after fixing merge conflict

**Pull request merged**
